crates/bitwarden/src/mobile/vault/client_ciphers.rs

```diff
@@ -29,6 +29,17 @@ impl<'a> ClientCiphers<'a> {
 
         Ok(CipherDecryptResponse { cipher })
     }
+
+    pub async fn decrypt_list(
+        &self,
+        req: CipherDecryptListRequest,
+    ) -> Result<CipherDecryptListResponse> {
+        let enc = self.client.get_encryption_settings()?;
+
+        let ciphers = req.ciphers.decrypt(enc, &None)?;
```

Comment:
Does this understand itself which decrypt interface to use?

Reply:
Yeah, the implementation for Vec just maps over the ciphers and calls the decrypt implementation of Cipher. Org id is ignored and each cipher uses its own.

https://github.com/bitwarden/sdk/blob/35c3c66d7bf7992624cb125688158553891f1437/crates/bitwarden/src/crypto.rs#L407-L412